serverless-now/serverless.yml

```diff
@@ -35,14 +55,14 @@ provider:
         - kms:Decrypt
       Resource:
         - "arn:aws:kms:ap-southeast-2:416527880812:key/3cf7aeb4-ad8c-4505-a8e3-7d2a556e188d"
-  alb:
+  alb: # Don't know what this stands for
     authorizers:
-      cognito-authorizer-uoapool:
+      uoapoolAuth:
         type: 'cognito'
         userPoolArn: 'arn:aws:cognito-idp:ap-southeast-2:416527880812:userpool/ap-southeast-2_pgErjyL4O'
         userPoolClientId: 'lrju6v80vse4bbaesjvnr2ff0' # required
         userPoolDomain: 'uoapool-sandbox' # required
```

Comment:
Why is this hardcoded and not in `env` files too?

Reply:
That code was intended to be a re-used authorizer but isn't yet working. The active/functioning authorizers are in the function section of the yml e.g. line 85-89 and 94-99. I just tested with that auth block commented out and it still denies posts without auth and accepts posts with auth. I can remove the auth if you'd like but it might be used in the future.